src/problem1.rb

```diff
@@ -0,0 +1,5 @@
+class Problem1
+  def sumOfAllEvenNumbers(values)
+    return values.select {|i| i.even?}.reduce(0, :+)
```

Comment:
Much better, 

Just out of curiosity why use `reduce` instead of `sum`?

Reply:
I used reduce because I read on ruby-doc that it applies a binary operation on all the elements of the array.